tensorflow_datasets/testing/imagewang.py

```diff
@@ -0,0 +1,81 @@
+# coding=utf-8
+# Copyright 2020 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+r"""Generate testing images for Imagewang. 
+
+Each image is a black 1 by 1 image.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+
+from absl import app
+from absl import flags
+import numpy as np
+import tensorflow as tf
+
+from tensorflow_datasets.core.utils import py_utils
+from tensorflow_datasets.testing import test_utils
+
+flags.DEFINE_string("tfds_dir", py_utils.tfds_dir(),
+                    "Path to tensorflow_datasets directory")
+FLAGS = flags.FLAGS
+
+_IMAGE_FILENAME = 'test_image'
```

Comment:
I think it would be better if the images had different file names styled similarly to what they're named in the dataset.

Reply:
Not sure about that. File names shouldn't make a difference I think.